### PR TITLE
Create FAQ.md

### PR DESCRIPTION
**Pull request by empeje**
## Type of changes

- :white_circle: Bug fix (Change that fixed an issue)
- :white_circle: Breaking change (Change that can cause existing functionality to change)
- :white_circle: Improvement (Change that improves the code. Maybe performance or development improvement)
- :white_circle: Feature (Change that adds new functionality)
- :white_circle: Documentation change (Change that modifies documentation. Maybe typo fixes)

## Checklist:

- :white_circle: My code follows [the project code style](docs/code_style.md)
- :white_circle: I have written test for my code and it has been tested
- :white_circle: All existing tests have been passed
- :white_circle: I have attached a screenshot/video to visualize my change if possible

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- [x] Create an awesome feature X
+- [ ] Do my homework
+
+</details>
+```
+
+</details>
+
+<details><summary>Other questions</summary>
```
Comment (Rokt33r) [inline FAQ.md]: Don't use `<details>` for this section. Let's leave a plain paragraph like "If you have other questions, [please create an issue][ISSUES]."

Comment (empeje) [reply FAQ.md]: sure

Comment (empeje) [reply FAQ.md]: updated @Rokt33r

Comment (Rokt33r) [reply FAQ.md]: Awesome!

**Comment by empeje** [comment]
Cool 👍

**Pull request merged**